app/presentation/representers/trip_representer.rb

```diff
@@ -16,8 +16,8 @@ class Trip < Roar::Decorator
       property :id
       property :query_id
       property :currency, extend: Representer::Currency, class: OpenStruct
-      property :origin                   # TODO: extend: Representer::Airport, class: OpenStruct
-      property :destination              # TODO: extend: Representer::Airport, class: OpenStruct
+      property :origin, extend: Representer::Airport, class: OpenStruct
+      property :destination, extend: Representer::Airport, class: OpenStruct
```

Comment:
Add the airport representer in ComfyWings::Representer::Flight origin and destination